x-pack/plugins/monitoring/public/components/cluster/overview/elasticsearch_panel.js

```diff
@@ -153,15 +154,22 @@ export function ElasticsearchPanel(props) {
   const showMlJobs = () => {
     // if license doesn't support ML, then `ml === null`
     if (props.ml) {
-      return [
-        <EuiDescriptionListTitle key="mlJobsListTitle">
-          <FormattedMessage
-            id="xpack.monitoring.cluster.overview.esPanel.jobsLabel"
-            defaultMessage="Jobs"
-          />
-        </EuiDescriptionListTitle>,
-        <EuiDescriptionListDescription key="mlJobsCount" data-test-subj="esMlJobs">{ props.ml.jobs }</EuiDescriptionListDescription>
-      ];
+      const gotoURL = `${chrome.getBasePath()}/app/ml#/jobs`;
```

Comment:
I don't think this is the link we want. We have a ML job listing page within monitoring, available at `/app/monitoring#/elasticsearch/ml_jobs`

Reply:
You're right, good catch!